features/cart/shopping_cart/maintaining_cart_after_authorization.feature

```diff
@@ -0,0 +1,26 @@
+@shopping_cart
+Feature: Maintaining cart after authorization
+    In order to not lose cart after authorization
+    As a Visitor
+    I want to be able to have my cart maintained
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Stark T-shirt" priced at "€12.00"
+        And there is user "robb@stark.com" identified by "KingInTheNorth"
+
+    @ui
+    Scenario: Having cart maintained after logging in
+        Given I have product "Stark T-Shirt" in the cart
+        When I log in as "robb@stark.com" with "KingInTheNorth" password
+        And I see the summary of my cart
```

Comment:
We should avoid using `see` IMO we can change it to simply `And I view my cart`

@TheMadeleine pick one: 
1. I view my cart
2. I browse my cart
3. I look into my cart 😆